tests/php/Extension/FluentSiteTreeExtensionTest.php

```diff
@@ -74,8 +75,8 @@ public function testGetLocaleInformation()
             $this->assertEquals('English (New Zealand)', $result->getTitle());
             $this->assertEquals('English', $result->getLanguageNative());
             $this->assertEquals('en', $result->getLanguage());
-            $this->assertEquals('/newzealand/a-page/', $result->getLink());
-            $this->assertEquals('http://mocked/newzealand/a-page/', $result->getAbsoluteLink());
+            $this->assertEquals(Controller::normaliseTrailingSlash('/newzealand/a-page/'), $result->getLink());
+            $this->assertEquals(Controller::normaliseTrailingSlash('http://mocked/newzealand/a-page/'), $result->getAbsoluteLink());
```

Comment:
Fixes failures which result from expecting a trailing slash when the default configuration is now to _not_ include one.

Doing it this way instead of just removing the trailing slash means the test is robust against this change. The test should be that the URL is correct and should not explicitly care if there's a trailing slash or not.